.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  # Update this to the name of the service you want to work with in your docker-compose.yml file
+  notify-ddapi:
```

Comment:
It works as is? i.e. you didn't have to define a `service` attribute in the `devcontainer.json` file? (I didn't try without it -- I guess it assumes it and use it correctly).

Reply:
Yeah it did work fine, but I didnt realize that was over there.  I will look more closely at the other repos and make this match.  Its too bad there is linting on these type of files since there are clearly dependencies between them!